README.md

```diff
@@ -275,7 +275,7 @@ To spawn multiple cursors from visual line/block modes type <kbd>ma</kbd>/<kbd>m
 
 See gif in action:
 
-![multicursors](/images/multicursor.gif)
+![multicursor](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/72b3f2b3-6d80-4ace-b6f4-b211baad74d1)
```

Comment:
Does this point to the image in https://github.com/vscode-neovim/vscode-neovim/issues/2015 ? How did you figure out the right URL? 

Please explain things instead of just merging PRs without explanation.

Reply:
I think this is a very common operation on GitHub, there's nothing much to explain. 
![msedge_yBDQUWqhUs](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/c756862a-b778-4840-a8c4-425365983f75)

Reply:
When I click on the image in that PR, it takes me to `https://private-user-images.githubusercontent.com/47070852/...`, which is a temporary URL that expires. But now I see that *editing* the comment does indeed expose the permanent URL. Hopefully that continues to work.

In any case, it's not obvious where the image comes from, and how to update it.